### PR TITLE
Sample code/improvement test

### PR DESCRIPTION
**Pull request by Saurabh186**

### CODE REVIEW

File: src/components/endpoints/codeTemplate.jsx [hunk 1]
```diff
@@ -109,15 +120,16 @@ class CodeTemplate extends Component {
                     this.makeCodeTemplate(key)
                   }}
                 >
-                  <img src={languages[key].imagePath} alt={languages[key].name} />
+                  <img src={languages[key].imagePath} alt={languages[key].name} width={15} />
                   {languages[key].name}
                 </button>
               ))}
-              <button className='codeTemplateButtonMore  d-flex justify-content-center align-items-center p-0'>
+              {/* <button className='codeTemplateButtonMore  d-flex justify-content-center align-items-center p-0'> */}
```
Comment (goutam01mehta) [inline src/components/endpoints/codeTemplate.jsx:127]: remove this code

File: src/components/endpoints/codeTemplate.jsx [hunk 2]
```diff
@@ -137,7 +149,7 @@ class CodeTemplate extends Component {
                     ))}
                   </Dropdown.Menu>
                 </Dropdown>
-              </button>
+              {/* </button> */}
```
Comment (goutam01mehta) [inline src/components/endpoints/codeTemplate.jsx:152]: ...

File: src/components/endpoints/codeTemplate.jsx [hunk 3]
```diff
@@ -11,7 +11,17 @@ import { languages, primaryLanguages, secondaryLanguages } from './languages'
 import { RiCloseLine } from "react-icons/ri";
 import { RxCopy } from "react-icons/rx";
 import IconButton from '../common/iconButton'
-
+import { BsThreeDotsVertical } from "react-icons/bs"
+export function hexToRgb(hex, opacity) {
```
Comment (goutam01mehta) [inline src/components/endpoints/codeTemplate.jsx:15]: this function is already in utility, export it

File: src/components/endpoints/codeTemplate.jsx (hunk 1 above)
Comment (Saurabh186) [reply src/components/endpoints/codeTemplate.jsx:127]: done

File: src/components/endpoints/codeTemplate.jsx (hunk 2 above)
Comment (Saurabh186) [reply src/components/endpoints/codeTemplate.jsx:152]: done

**Review by goutam01mehta — APPROVED** [review]

**Pull request merged**